packages/ui/core/Button/types.tsx

```diff
@@ -0,0 +1,40 @@
+import {TouchableOpacityProps} from 'react-native';
+import {DefaultTheme} from 'styled-components/native';
+
+// gradient
+export const gradientConfig = {
+  start: {
+    x: 0,
+    y: 0,
+  },
+  end: {
+    x: 1,
+    y: 1,
+  },
+  locations: [0.1, 0.9],
+};
+
+// sizes
+export const LARGE = 'large';
+export const SMALL = 'small';
+
+export type ButtonActive = boolean;
```

Comment:
I'm not sure if you should create those types just to wrap primitives. I'd rather specify the prop color as string for instance.

Reply:
Me too, I have following the patterns of this project. What do you think @renanmav?

Reply:
Imagine that we need to specify the button color (green) in every usage of Button component. Then, imagine if we would like to change the button color to red instead. Every Button reference would need to be updated. 

But sure, we can have an optional `color` prop that, if wasn’t specified, matches to the default color.